### PR TITLE
Appends the chain_id to the metadata sent to the http solver.

### PR DESCRIPTION
**Pull request by marcovc**
To make it easier to debug things on the solver end.

### CODE REVIEW

File: solver/src/solver/http_solver.rs [hunk 1]
```diff
@@ -219,7 +219,11 @@ impl HttpSolver {
             orders: order_models,
             amms: amm_models,
             metadata: Some(MetadataModel {
-                environment: Some(self.network_id.clone()),
+                environment: Some(format!(
+                    "{} / {}",
+                    self.network_id.clone(),
+                    self.chain_id.clone()
```
Comment (nlordell) [inline solver/src/solver/http_solver.rs:225]: Note that for Mainnet, Rinkeby and xDai, `network_id == chain_id` AFAIK.

Comment (marcovc) [reply solver/src/solver/http_solver.rs:225]: I'm getting "Ethereum" as network_id currently. Are you saying I will get "Ethereum / Ethereum" for all of those networks with this change? If so I must really revise this :)

Comment (marcovc) [reply solver/src/solver/http_solver.rs:225]: I recall I coded [this](https://github.com/gnosis/gp-v2-services/blob/ae3d2387cf0bc464cab38fdffb62b3f3263d7165/shared/src/network.rs) a while ago. Maybe I could make it more informative and just use network_name here ?

Comment (marcovc) [reply solver/src/solver/http_solver.rs]: I changed the code to use that function instead.

**Review by josojo — APPROVED** [review]

File: solver/src/solver/http_solver.rs (hunk 1 above)
Comment (nlordell) [reply solver/src/solver/http_solver.rs]: To me, the network ID is the result of `net_version` RPC call and the chain ID is the result of the `eth_chainId` RPC call. They can be different, but for all networks we support they are the same (1, 4, and 100 respectively).

**Review by nlordell — APPROVED** [review]
Ahh, I see the issue `self.network_id` was not what I expected it to be. Looks good now!

**Comment by marcovc** [comment]
> Ahh, I see the issue self.network_id was not what I expected it to be. Looks good now!

Thanks, yes that should have been better named to avoid this kind of confusion. Also fixed now!

**Review by vkgnosis — APPROVED** [review]

**Pull request merged**